specs/pvs/corla.pvs

```diff
@@ -725,7 +793,10 @@ BEGIN
 
 % @review CDOS Must audit boards have a minimum number of members?
 % A maximum?
+% @trace Per email from Jerome Lovato at CDOS on 19 July 2017: The
+% audit board has exactly two members.
   audit_board: TYPE = set[audit_board_member]
+  audit_board_size: AXIOM TBD
```

Comment:
Yes, I capture that in the @trace annotation and will write the constraint, encoded in an axiom, later.  Note that `TBD` is simply a synonym for `true` at the moment.